### PR TITLE
Feedback System/Tracking

### PR DESCRIPTION
**Pull request by jeffdechambeau**
- Wired nagging back in
- Fixed csv to play nice with pkg
- Simplified buttons
- Fixed meetingEnd followup
- Fixed unexpected button behaviours

### CODE REVIEW

File: src/csv.js [hunk 1]
```diff
@@ -19,7 +18,7 @@ async function ensure() {
                     'success',
                 ],
             })
-        if (!data.length) throw "fix"
+        if (!data.length) throw 'fix'
```
Comment (jeznag) [inline src/csv.js:21]: Can we add more details to this assertion?

File: src/calendar/tidy.js [hunk 2]
```diff
@@ -39,3 +39,24 @@ function tidyEvent(evt) {
...
+            url: null,
+            id: 'F2DBD832-EF8B-4608-93F2-C000EC1B9ABA',
+        },
+    ]
+    const evt1 = {
+        description: `https://teams.microsoft.com///meetup-join/19%3ameeting_ZmYwTY4YjctMzMxMi00MmU5LWFjYTAtNiMwMzE5NZM4Y2F1%40thread.v2/0?context=%7b%22Tid%22%3a%223b518aae-8921-4a7b-8497-619d756c20%22%2%220id%22%33%22b97e1b4b-edb5-48eb-a7830b64a66a55333%22%7d>`,
```
Comment (jeznag) [inline src/calendar/tidy.js:58]: It shouldn't include the > at the end

Comment (jeznag) [reply src/calendar/tidy.js:58]:
```
const { matchService } = require('../calendar/tidy')

const testCases =
    [
        {
            calendarData: { "summary": "SMS for Zoho Standup", "calendarName": "Calendar", "startDate": "2023-01-13T05:30:00.000Z", "endDate": "2023-01-13T05:45:00.000Z", "location": null, "description": "\r\n________________________________________________________________________________\r\nMicrosoft Teams meeting\r\nJoin on your computer or mobile app\r\nClick here to join the meeting<https://teams.microsoft.com/l/meetup-join/19%3ameeting_ZmYwYTY4YjctMzMxMi00MmU5LWFjYTAtNjMwMzE5NzM4Y2Fl%40thread.v2/0?context=%7b%22Tid%22%3a%223b518aae-8921-4a7b-8497-619d756ce20e%22%2c%22Oid%22%3a%22b97e1b4b-edb5-48eb-a783-b64a66a55333%22%7d>\r\nOr call in (audio only)\r\n+1 412-912-4129,,758276296#<tel:+14129124129,,758276296#>   United States, Pittsburgh\r\nPhone Conference ID: 758 276 296#\r\nFind a local number<https://dialin.teams.microsoft.com/9b869321-e683-4bf6-ac06-e2c52d67cb9a?id=758276296> | Reset PIN<https://mysettings.lync.com/pstnconferencing>\r\nLearn more<https://aka.ms/JoinTeamsMeeting> | Meeting options<https://teams.microsoft.com/meetingOptions/?organizerId=dca1ad50-1d0d-449f-a28d-8929ffbf7eaa&tenantId=4ae1b72f-a288-4b5d-98ad-dbe05c630a89&threadId=19_meeting_ZmYwYTY4YjctMzMxMi00MmU5LWFjYTAtNjMwMzE5NzM4Y2Fl@thread.v2&messageId=0&language=en-GB>\r\n________________________________________________________________________________\r\n", "url": "https://teams.microsoft.com/l/meetup-join/19%3ameeting_ZmYwYTY4YjctMzMxMi00MmU5LWFjYTAtNjMwMzE5NzM4Y2Fl%40thread.v2/0?context=%7b%22Tid%22%3a%223b518aae-8921-4a7b-8497-619d756ce20e%22%2c%22Oid%22%3a%22b97e1b4b-edb5-48eb-a783-b64a66a55333%22%7d>", "id": "BD834825-C84B-48EB-9B3E-5B531ED15F6D" },
            expectedUrl: 'https://teams.microsoft.com/l/meetup-join/19%3ameeting_ZmYwYTY4YjctMzMxMi00MmU5LWFjYTAtNjMwMzE5NzM4Y2Fl%40thread.v2/0?context=%7b%22Tid%22%3a%223b518aae-8921-4a7b-8497-619d756ce20e%22%2c%22Oid%22%3a%22b97e1b4b-edb5-48eb-a783-b64a66a55333%22%7d',

        }
    ];

testCases.forEach((testCase) => {
    const actualResult = matchService(testCase.calendarData);

    if (actualResult.url !== testCase.expectedUrl) {
        throw new Error(`Expected ${testCase.expectedUrl} - actual ${actualResult.url}`);
    }
})
```

**Comment by jeznag** [comment]
CleanShot 2023-02-07 at 07 45 26

Barks don't stop when I click "I don't need to attend"

File: src/calendar/tidy.js (hunk 2 above)
Comment (jeffdechambeau) [reply src/calendar/tidy.js:68]: Addressed in latest commit

File: src/csv.js (hunk 1 above)
Comment (jeffdechambeau) [reply src/csv.js]: Done

**Comment by jeffdechambeau** [comment]
> 

Fixed, I missed exporting the manually specified dialog timeout length from config.js, was trying to wait for NaN seconds.

**Pull request merged**